python/lsst/dm/rucio/register/rucio_interface.py

```diff
@@ -111,8 +111,10 @@ def _make_did(self, resource_path, metadata: str) -> RucioDID:
             adler32 = f"{zlib.adler32(contents):08x}"
         path = resource_path.unquoted_path.removeprefix(self.rse_root)
         pfn = self.pfn_base + path
-        logging.debug(f"pfn = {pfn}")
-        name = path.removeprefix("/")
+        logging.debug(f"{pfn=}")
+        name = path.removeprefix("/" + self.scope + "/")
```

Comment:
This looks correct now.  It might be a little safer to try to use `butler.get_datastore_roots()` to get the prefix to remove rather than regenerate it from `rse_root + scope`.

Reply:
I'll keep this in mind for the next update.